# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/15 15:43:47-08:00 adaplas@hotpop.com 
#   [PATCH] radeonfb: Fix init/exit section usage
#   
#     Fix 3 instances of calling exit section function from text or init:
#     Error: ./drivers/video/aty/radeon_base.o .text refers to 000029e5
#     R_386_PC32        .exit.text
#     Error: ./drivers/video/aty/radeon_base.o .text refers to 00002a60
#     R_386_PC32        .exit.text
#     Error: ./drivers/video/aty/radeon_base.o .init.text refers to 00000192
#     R_386_PC32        .exit.text
#   
#   Signed-off-by: Randy Dunlap <rddunlap@osdl.org>
#   Signed-off-by: Antonino Daplas <adaplas@pol.net>
#   Signed-off-by: Andrew Morton <akpm@osdl.org>
#   Signed-off-by: Linus Torvalds <torvalds@osdl.org>
# 
# drivers/video/aty/radeon_base.c
#   2005/01/15 14:31:04-08:00 adaplas@hotpop.com +1 -1
#   radeonfb: Fix init/exit section usage
# 
diff -Nru a/drivers/video/aty/radeon_base.c b/drivers/video/aty/radeon_base.c
--- a/drivers/video/aty/radeon_base.c	2005-01-25 02:32:03 -08:00
+++ b/drivers/video/aty/radeon_base.c	2005-01-25 02:32:03 -08:00
@@ -261,7 +261,7 @@
 
 #endif /* CONFIG_PPC_OF */
 
-static void __devexit radeon_unmap_ROM(struct radeonfb_info *rinfo, struct pci_dev *dev)
+static void radeon_unmap_ROM(struct radeonfb_info *rinfo, struct pci_dev *dev)
 {
 	if (!rinfo->bios_seg)
 		return;
